qiskit_ibm_runtime/qiskit/primitives/base_estimator.py

```diff
@@ -0,0 +1,230 @@
+# This code is part of Qiskit.
```

Comment:
This is copied from terra. Don't review this file. Will be removed from here when terra code is released.